api/controllers/calendarEntry.controller.ts

```diff
@@ -1,34 +1,36 @@
-import { Request, Response, NextFunction } from 'express';
-import { db } from '../app'
-
+import { Request, Response, NextFunction } from "express";
+import { CalendarEntry } from "../models/calendarEntry";
+import { dayAfter } from "../lib/dateHelpers";
 
 interface CalendarEntryInput {
-  eventId: string
-  creatorId: string
-  title: string
-  description: string
-  isAllDay: boolean
-  startTimeUtc: Date
-  endTimeUtc: Date
-};
+  eventId: string;
+  creatorId: string;
+  title: string;
+  description: string;
+  isAllDay: boolean;
+  startTimeUtc: Date;
+  endTimeUtc: Date;
+}
```

Comment:
@elliedori @sofita @angepol @Jake-Mitchell It would be great to sync up when we all have a chance to configure our code formatting settings to be the same. that way we don't have the reformatting to parse through along with the rest of the changes in the PR.